graylog2-web-interface/src/components/common/DropdownMenu.tsx

```diff
@@ -27,9 +27,9 @@ type Props = {
 };
 
 const StyledDropdownMenu = styled.ul.attrs(() => ({
-  className: 'dropdown-menu', /* stylelint-disable-line property-no-unknown */
+  className: 'dropdown-menu' /* stylelint-disable-line property-no-unknown*/
 }))<{ $show: boolean, $zIndex: number }>(({ $show, theme, $zIndex }) => css`
-  display: ${$show ? 'block' : 'none'};
+  display: ${$show ? "block" : "none"};
```

Comment:
We should favor single quotes consistently instead.

Reply:
I had the same thought, I adjusted the stylelint rule with https://github.com/Graylog2/graylog2-server/pull/14480 already, but I wanted to fix all the other auto-fixable hints first.